### PR TITLE
Add strange-room/room.md

### PR DESCRIPTION
**Pull request by chilito11**
Please review this, @casey-collab

### CODE REVIEW

File: english/strange-room/room.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+You hear a strange noise then turn and see a large, wooden door.
```
Comment (casey-collab) [inline english/strange-room/room.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by chilito11** [comment]
Please review this, @casey-collab

I added a reference to strange-room/room.md per your guideance. Thanks in advance!

**Comment by casey-collab** [comment]
Looks great! :octocat:

**Pull request merged**